app/main/posts/modify/post-tabs.directive.js

```diff
@@ -48,11 +48,10 @@ function PostVerticalTabsController(
     }
 
     function stageIsComplete(stageId) {
-        return _.includes($scope.post.completed_stages, stageId);
+        return $scope.post.completed_stages.indexOf(stageId) > -1;
```

Comment:
This doesn't fix the bug. It still both fails to save the stage as completed, and fails post validation